src/TextField/TextField.jsx

```diff
@@ -24,6 +24,9 @@ function isValid(value) {
 const TextField = React.createClass({
 
   propTypes: {
+    /**
+     * @ignore
```

Comment:
Although interestingly enough we had a case yesterday of someone using `children` as a prop, rather than encliosng the children. (Maybe that actually works - I have no idea!) 

In either case we still need some way to identify that a component accepts children.